docs/developers/plugins/index.rst

```diff
@@ -61,3 +61,14 @@ Available plugin components
    appointment_plugins
    payment_plugins
 
+External resources
+==================
+
+Microsoft Graph
+---------------
+
+Microsoft graph is used in one of the core registration plugins.
+
+Microsoft graph has a web tool that lets you play with the API either with a sample account, or with your own microsoft
+account. This tool can be found `here <https://developer.microsoft.com/en-us/graph/graph-explorer>`_.
+
```

Comment:
maybe it'd be nicer to set up an additional page under https://open-forms.readthedocs.io/en/latest/developers/backend/index.html#development-and-debug-tooling for External resources/tools and put it there.

When we eventually add documentation for the registration plugins, we can them simply refer to this subtitle with a `:ref:` role.